### ISSUE TITLE
Field names are wrongly passed as kwarg to built-in bulk_update method

### ISSUE THREAD

**Issue by 03b8** [Tesorio/django-anon#63]
## Checklist

- [ ] I have verified that that issue exists against the `master` branch of django-anon
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [x] I have reduced the issue to the simplest possible case
- [ ] I have included a failing test as a pull request (If you are unable to do so we can still accept the issue)

## Steps to reproduce

It seems like in some cases (I can't tell on what basis), an Anonymizer will attempt to execute the built-in Django `bulk_update` while passing it the list of field names as a keyword argument, which then results in a `TypeError: bulk_update() got an unexpected keyword argument 'update_fields'`. I doubt this is helpful at all, but here is one of the Anonymizers which leads to the exception being thrown:

```python
from faker import Faker

fake = Faker()

class DemandDocumentAnonymizer(anon.BaseAnonymizer):
    description = fake.text
    document_url = fake.url
    instructions = fake.text

    @lazy_attribute
    def name(self):
        return fake.file_name(extension="pdf")

    class Meta:
        model = app.models.ExampleModel
```
## Expected behavior
The [built-in `bulk_update()` method](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-update) receives the fields to be updated as a positional argument.
## Actual behavior
In some cases (unclear which ones) the [built-in `bulk_update()` method](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-update) receives the fields to be updated as a keyword argument (`update_fields`).

**Comment by caioariede** [comment]
@03b8 can you please confirm which version of django-anon you're using?

**Comment by caioariede** [comment]
@03b8 this issue should be fixed with [version 0.3.1](https://pypi.org/project/django-anon/0.3.1/)

**Comment by 03b8** [comment]
nice! thank you for the quick response and the fix.
